hazelcast/src/main/java/com/hazelcast/internal/partition/PartitionRuntimeState.java

```diff
@@ -80,55 +87,47 @@ public PartitionRuntimeState(InternalPartition[] partitions, Collection<Migratio
     }
 
     /** Create a map from address to the replica index on the first partition where the address is a replica */
```

Comment:
Seems I produced a wrong javadoc here. We can remove it in a follow-up PR.

Reply:
I'll update that.